test.opt

```diff
@@ -0,0 +1 @@
+ALL XFAIL gnat2goto fails with private type declarations unsupported
```

Comment:
Where is this file?

Reply:
Yes, this seems to be misplaced?

Reply:
I think I removed it because we know support private types?

Reply:
It's not whether or not was it removed, but this file is now stored in `gnat2goto/test.opt`, i.e. the top folder. There's no purpose for it there.

Reply:
As the other reviewers mentioned, this file should not be in this location. If it's meant to be removed, please remove it from the commit (`git rm`)